plugins/telemetry/vppcalls/telemetry_handler_api.go

```diff
@@ -160,7 +160,7 @@ var Handler = vpp.RegisterHandler(vpp.HandlerDesc{
 type NewHandlerFunc func(govppapi.Channel) TelemetryVppAPI
 
 // AddHandlerVersion registers vppcalls Handler for the given version.
-func AddHandlerVersion(version string, msgs []govppapi.Message, h NewHandlerFunc) {
+func AddHandlerVersion(version vpp.Version, msgs []govppapi.Message, h NewHandlerFunc) {
 	Handler.AddVersion(vpp.HandlerVersion{
```

Comment:
there is a large commented block of code in `func CompatibleTelemetryHandler(c vpp.Client) TelemetryVppAPI` inside this file. Is it still needed?